site/community/Community-Resources.md

```diff
@@ -101,3 +103,4 @@ To explore other community-developed resources and content about GraphQL, take a
 - [Learn Apollo](https://www.learnapollo.com/): A hands-on tutorial for Apollo GraphQL Client (for React, ReactNative, Exponent, Angular, Vue & iOS)
 - [Learn Relay](https://www.learnrelay.org): A comprehensive introduction to Relay (includes video tutorials)
 - [Scaphold Community](https://scaphold.io/community/): Resources to help you launch a production app with GraphQL
+- [GraphQL World](https://graphql-world.com) - The fastest growing community of GraphQL developers
```

Comment:
Can you switch the hyphen out for a colon please, to match the other entries here?